libraries/AP_Scripting/applets/trisonica-mini.lua

```diff
@@ -0,0 +1,100 @@
+-- Script decodes and logs wind sensor data for Trisonica LI-550 mini
+-- https://www.licor.com/products/trisonica/LI-550-mini
+
+-- Parameters:
+-- SCR_ENABLE 1
+-- SERIAL5_PROTOCOL 28
+
+-- In SITL, you can enable serial ports to connect to the real device.
+-- https://ardupilot.org/dev/docs/learning-ardupilot-uarts-and-the-console.html#the-8-uarts
+-- ./Tools/autotest/sim_vehicle.py -v Plane --console --map -A "--serial5=uart:/dev/ttyUSB0" -D
+
+-- Remember to change baud to 230k in the sensor setup and enable the fields you want.
+-- Also, enable 10Hz output instead of the default 5Hz.
+
+-- Example data string (excluding quotes, including the carriage return line feed ending)
```

Comment:
What precisely is the ending? As you describe it's "\r\n" but the code doesn't match that and no endings are in the sample.

Reply:
`\r\n`. I just matched `\r` because it was easy.